x-pack/plugins/maps/public/classes/layers/blended_vector_layer/blended_vector_layer.test.tsx

```diff
@@ -94,6 +95,7 @@ describe('getSource', () => {
         indexPatternId: 'myIndexPattern',
         applyGlobalQuery: false,
         applyGlobalTime: false,
+        applyForceRefresh: true,
```

Comment:
Great, this test caught a regression. applyForceRefresh is expected to be false. The reason being is that the cluster source should set these values from the document source and that is not happening for applyForceRefresh.

To fix the problem:
1) in blended_vector_layer.test.tsx after line 65, add `applyForceRefresh: false,`
2) in blended_vector_layer.tsx after line 62, add `clusterSourceDescriptor.applyForceRefresh = documentSource.getApplyForceRefresh();`

Reply:
thx for flagging the missing property copy.